bundle-size/app.js

```diff
@@ -68,6 +70,44 @@ function formatBundleSizeDelta(delta) {
   return 'Δ ' + (delta >= 0 ? '+' : '') + delta.toFixed(2) + 'KB';
 }
 
+/**
+ * Returns an encouraging result title when the bundle size is reduced.
+ * Otherwise the title is neutral, but not discouraging.
+ *
+ * @param {number} delta bundle size delta.
+ * @param {string} deltaFormatted bundle size delta from `formatBundleSizeDelta`
+ * @return {string} successful title message.
+ */
+function successfulTitleMsg(delta, deltaFormatted) {
+  const noApprovalNecessaryDesc = 'no approval necessary';
+  if (delta <= HUMAN_ENCOURAGEMENT_MAX_DELTA) {
+    return `${deltaFormatted} 🎉 | ${noApprovalNecessaryDesc}`;
+  }
+  return `${deltaFormatted} | ${noApprovalNecessaryDesc}`;
+}
+
+/**
+ * Returns an encouraging result summary when the bundle size is reduced.
+ * Otherwise the summary is neutral, but not discouraging.
+ *
+ * @param {number} delta bundle size delta.
+ * @param {string} deltaFormatted bundle size delta from `formatBundleSizeDelta`
+ * @return {string} successful summary message.
+ */
+function successfulSummaryMsg(delta, deltaFormatted) {
+  const sizeDesc = 'gzipped compressed size of `v0.js`';
+  const deltaDesc = `The bundle size change is ${deltaFormatted}.`;
+
+  if (delta <= HUMAN_ENCOURAGEMENT_MAX_DELTA) {
+    return `This pull request *reduces* the bundle size (${sizeDesc}), so no ` +
```

Comment:
I don't mind just having that text in `sizeDesc` and `deltaDesc` repeated twice, since it's easier for readers of the source code to read this instead of jumping up and down, and I feel like readability here is more important than strict DRI